### PR TITLE
Bug Fix: Stop file template variables from persisting over pages

### PR DESCRIPTION
**Pull request by Ken-Lo**

### CODE REVIEW

**Comment by Ken-Lo** [comment]
@harrison Turns out the [_.extend method](http://underscorejs.org/#extend) will copy over methods from the `fileMetadata` object which then persist in the `globalMetadata` object for all subsequent page renders.
Fix: Define a new `metadata` object in which `globalMetadata` and `fileMetadata` are merged into.

File: scripts/nunjucks.js [hunk 1]
```diff
@@ -107,7 +107,8 @@ findFiles((argv.i || argv.input), function(filepath) {
   console.log('Compiling %s (%s)', filepath, outfile);
 
   stream(filepath, outfile, function(file, fileMetadata) {
-    var metadata = _.extend(globalMetadata, fileMetadata);
```
Comment (samjewell) [inline scripts/nunjucks.js]: you could also write:
`var metadata = _.extend({}, globalMetadata, fileMetadata);`

Comment (harrison) [inline scripts/nunjucks.js]: :+1: `var metadata = _.extend({}, globalMetadata, fileMetadata);`

**Comment by harrison** [comment]
Nice! :+1: after comments

**Comment by Ken-Lo** [comment]
@harrison @samjewell Thanks guys, modified as suggested!

**Pull request merged**